stemcell_builder/stages/bosh_azure_agent_settings/apply.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+base_dir=$(readlink -nf $(dirname $0)/../..)
+source $base_dir/lib/prelude_apply.bash
+
+# Set SettingsPath but never use it because file_meta_service is avaliable only when the settings file exists.
+cat > $chroot/var/vcap/bosh/agent.json <<JSON
+{
+  "Platform": {
+    "Linux": {
+      "DevicePathResolutionType": "virtio"
+    }
+  },
+  "Infrastructure": {
+    "NetworkingType": "dhcp",
```

Comment:
We have set use_dhcp in Azure CPI https://github.com/Azure/bosh-azure-cpi-release/blob/master/src/bosh_azure_cpi/lib/cloud/azure/cloud.rb#L416. I will remove this from agent.json as AWS did https://github.com/cloudfoundry/bosh/commit/2f95fd89b8bbb1edad8e9e52d6f80d5996acd629.